app/views/questionnaire.py

```diff
@@ -644,14 +644,28 @@ def _get_context(full_routing_path, block, current_location, form):
     if block['type'] == 'Summary':
         rendered_schema_json = renderer.render(g.schema.json, **schema_context)
         form = form or FlaskForm()
-        summary_rendered_context = build_summary_rendering_context(g.schema, rendered_schema_json, answer_store, metadata)
+        summary_rendered_context = build_summary_rendering_context(g.schema, rendered_schema_json, answer_store,
+                                                                   metadata)
```

Comment:
why?

Reply:
PEP8 error complaining about line length

Reply:
but the length hasn't changed so why has this started erroring?

Reply:
Oh no its always been there just no ones bothered to move it to a new line, similar instances of this are scattered throughout the code base as well